packages/blockchain-link-utils/src/solana.ts

```diff
@@ -35,7 +36,18 @@ export const transformTokenInfo = (
                     ...getTokenNameAndSymbol(info.mint, tokenDetailByMint),
                 };
             }),
-            A.uniqBy(token => token.contract),
+            A.reduce({}, (acc: { [mint: string]: TokenInfo }, token: TokenInfo) => {
```

Comment:
Added this to account for the fact that users can potentially create an arbitrary amount of token accounts to hold a given token. We should aggregate the balance of all these accounts and handle sending from them too in a latter PR.